### ISSUE TITLE
About the 1.7.10 latest version

### ISSUE THREAD

**Issue by vinicius1222** [Tslat/Advent-Of-Ascension#3960]
Hi, I know that version 1.7.10 is no longer valid, but with great difficulty I managed to solve the bugs that were in the last version, don't worry, I only have the corrected file, I really like version 1.7. 10 because there are mods that only exist up to this version, and as it was, you couldn't play, and the older versions from 2.4 onwards have a lot of bugs.

**Comment by Tslat** [comment]
1.7.10 was perfectly playable

**Comment by vinicius1222** [comment]
Sure, is perfectly playable, I just can't use the morph mods, hats, blood magic and any other mods that save data in a specific location in the file that contains the players' information because they will be reset every time they leave the world or disconnect from the server. Apart from the fact that the bane boss doesn't summon the minions completely, leaving them as buggy entities that can't hit or do anything. I wanted to say that I fixed that, I did it, I had to do it.
And I didn't send this to anyone, because I'm still testing it, and also because this mod isn't mine to send changes on my part without authorization from the owner, in this case yours.
